pkgs/tools/archivers/bomutils/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bomutils";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "hogliux";
+    repo = pname;
+    rev = version;
+    sha256 = "1i7nhbq1fcbrjwfg64znz8p4l7662f7qz2l6xcvwd5z93dnmgmdr";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/hogliux/bomutils";
+    description = "Open source tools to create bill-of-materials files used in Mac OS X installers";
+    platforms = platforms.all;
+    license = licenses.gpl2;
```

`gpl2` is deprecated, use `gpl2Only` or gpl2Plus`. The source files give the option of a later version, so the latter is appropriate.

```suggestion
    license = licenses.gpl2Plus;
```